locale/update.sh

```diff
@@ -10,10 +10,10 @@ set -ex
 
 # pull po files from transifex
 cd `dirname $0`
-sphinx-intl create-transifexrc
+sphinx-intl create-transifexrc --transifex-token=$TX_TOKEN
```

Comment:
@shimizukawa Could you check that you set `TX_TOKEN` variable in settings?

Reply:
OK, I'll take a look.

Reply:
Currently, we use `SPHINXINTL_TRANSIFEX_PASSWORD` environment variable. 
Now I add `TX_TOKEN` envval and set it a token.

Reply:
I think `SPHINXINTL_TRANSIFEX_TOKEN` is better than `TX_TOKEN` because `SPHINXINTL_TRANSIFEX_TOKEN` is used by sphinx-intl without `--transifex-token` commandline option.